src/Core/Framework/DataAbstractionLayer/FilteredEntityExtension.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\DataAbstractionLayer;
+
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * EntityDefinitions allow only regular EntityExtension, this class maps BulkyEntityExtensions to EntityExtensions
+ *
+ * @internal
+ */
+#[Package('core')]
+class FilteredEntityExtension extends EntityExtension
```

Comment:
```suggestion
class FilteredBulkEntityExtension extends EntityExtension
```

IMHO the current name is too generic